### PR TITLE
RSDK-808 extra params for movement sensor

### PR DESCRIPTION
**Pull request by maximpertsov**
JIRA: https://viam.atlassian.net/browse/RSDK-808

* Merge before
  * [x] Sensor updates: https://github.com/viamrobotics/rdk/pull/1550
  * [x] API updates: https://github.com/viamrobotics/api/pull/91

### CODE REVIEW

File: components/movementsensor/movementsensor.go [hunk 1]
```diff
@@ -48,22 +48,26 @@ func init() {
 			Lat float64
 			Lng float64
 		}
-		p, _, err := ms.Position(ctx)
+		// TODO: should collectors pass in extra parameters?
```
Comment (cheukt) [inline components/movementsensor/movementsensor.go:51]: i don't think we need to right now

**Review by cheukt — APPROVED** [review]

File: services/navigation/builtin/builtin.go [hunk 2]
```diff
@@ -169,7 +169,9 @@ func (svc *builtIn) startWaypoint() error {
 				return
 			}
 
-			currentLoc, _, err := svc.movementSensor.Position(svc.cancelCtx)
+			// TODO: once https://github.com/viamrobotics/rdk/pull/1545 is merged,
```
Comment (stuqdog) [inline services/navigation/builtin/builtin.go:172]: this is already merged, probably should update here.

File: services/navigation/builtin/builtin.go [hunk 3]
```diff
@@ -247,7 +249,9 @@ func (svc *builtIn) Location(ctx context.Context, extra map[string]interface{})
 	if svc.movementSensor == nil {
 		return nil, errors.New("no way to get location")
 	}
-	loc, _, err := svc.movementSensor.Position(ctx)
+	// TODO: once https://github.com/viamrobotics/rdk/pull/1545 is merged,
```
Comment (stuqdog) [inline services/navigation/builtin/builtin.go:252]: this is already merged, probably should update here.

File: services/navigation/builtin/builtin.go (hunk 2 above)
Comment (maximpertsov) [reply services/navigation/builtin/builtin.go:172]: ah great, will do

**Pull request merged**